src/vs/workbench/contrib/mergeEditor/browser/view/editors/inputCodeEditorView.ts

```diff
@@ -376,3 +390,7 @@ export class MergeConflictGutterItemView extends Disposable implements IGutterIt
 		});
 	}
 }
+
+function clamp(value: number, min: number, max: number): number {
+	return Math.min(Math.max(value, min), max);
+}
```

Comment:
This already exists:

https://github.com/microsoft/vscode/blob/114180866982a3bf28aba18fd9ddd581aeb3612b/src/vs/base/common/numbers.ts#L6-L8